mmv1/templates/terraform/encoders/compute_backend_bucket.go.erb

```diff
@@ -0,0 +1,35 @@
+c, cdnPolicyOk := d.GetOk("cdn_policy")
+
+if !cdnPolicyOk {
+  return obj, nil
+}
+
+cdnPolicies := c.([]interface{})
+
+if len(cdnPolicies) == 0 {
+  return obj, nil
+}
+
+cdnPolicy := cdnPolicies[0].(map[string]interface{})
+
+cacheMode := cdnPolicy["cache_mode"].(string)
+_, defaultTTLOk := cdnPolicy["default_ttl"]
+_, clientTTLOk := cdnPolicy["client_ttl"]
```

Comment:
```suggestion
_, defaultTTLOk := cdnPolicy["default_ttl"]
_, clientTTLOk := cdnPolicy["client_ttl"]
```
These seem irrelevant to me. It doesn't matter what is in state here. It only matters what is in the `obj` or what we are sending to the api.